app/Http/Controllers/Api/Auth/LoginController.php

```diff
@@ -2,6 +2,80 @@
 
 namespace App\Http\Controllers\Api\Auth;
 
-class LoginController
+use App\Http\Controllers\Controller;
+
+class LoginController extends Controller
```

Comment:
In my opinion is better do this more SOLID and split each action to a custom controller. But we can refactor later

Reply:
its true, you are right.